jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/RuntimeDataServiceImplTest.java

```diff
@@ -1152,6 +1151,7 @@ public void testGetTaskAssignedAsBusinessAdminByStatus() {
         List<TaskSummary> tasks = runtimeDataService.getTasksAssignedAsBusinessAdministratorByStatus("Administrator", statuses, new QueryFilter(0, 5));
         assertNotNull(tasks);
         assertEquals(5, tasks.size());
+        assertTasks(tasks, 5, pids);
```

Comment:
this method is checking for the correlationKey rather than pids. Probably in this case correlationKey = pid always? would it not make sense to modify `assertTasks` method to fetch pids instead of correlationKey? all the calls to this method are passing pids. What do you think?

Reply:
the goal for this assertTasks method is to verify that correlationKey and processType have the expected value (_pids_ and _process_ respectively), as it was introduced in the [jbpm#1680](https://github.com/kiegroup/jbpm/pull/1680)